### ISSUE TITLE
Raising PropertyChanged Event for Non-BusinessObject

### ISSUE THREAD

**Issue by CrossSlide** [MarimerLLC/cslaforum#500]
I have added a Property to my ViewModel that is not related to my BusinessObject. How do I raise the PropertyChanged event for this property to update my View?

**Comment by jonnybee** [comment]
https://docs.microsoft.com/en-us/dotnet/framework/winforms/how-to-implement-the-inotifypropertychanged-interface

**Comment by CrossSlide** [comment]
Thanks @jonnybee. I was using that link but when I create the PropertyChanged event handler it says I'm hiding the inherited ViewModelBase.PropertyChanged. So how is this done when the ViewModelBase already implements INotifyPropertyChanged?

**Comment by hurcane** [comment]
Yes, it is a common pattern to have a protected OnPropertyChanged method available for subclasses to invoke an event defined in a base class. It's even documented as an official technique by Microsoft: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/events/how-to-raise-base-class-events-in-derived-classes

**Comment by CrossSlide** [comment]
Thanks @hurcane!

**Comment by jonnybee** [comment]
Call OnPropertyChanged("propertyName"); This method exists in the base class.

**Comment by CrossSlide** [comment]
Thanks @jonnybee!
